transformations/chinese_antonym_synonym_substitution/test.json

```diff
@@ -0,0 +1,62 @@
+{
+  "type": "chinese_antonym_synonym_substitution",
+  "test_cases": [
+    {
+      "class": "ChineseAntonymAndSynonymSubtitution",
+      "inputs": {
+        "sentence": "汉字是语素文字，总数非常庞大。汉字总共有多少字？到目前为止，恐怕没人能够答得上来精确的数字。"
+      },
+      "outputs": [{
+        "sentence": "汉字是语素言，总额深深的大幅度。单字合计有些微字？到目前为止，恐没人能够答得上来毫厘不爽的数字。"
```

Comment:
seems like some characters are dropped: 恐怕没人 => 恐没人

Reply:
It looks like this is a weakness of the library that is being used for the synonym substitution where it will drop words if the probability is high, I have added this to the README.